src/app/shared/date/timeago.pipe.ts

```diff
@@ -0,0 +1,35 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
```

Comment:
No timeago spec?

Reply:
There are specs for it in publish and I was trying to minimize how many files I brought over 100% identically from publish ... (I now realize that's a case in point for "put it in styleguide".). OK to put specs in styleguide version of it as part of https://github.com/PRX/styleguide.prx.org/issues/80 ?

Reply:
sounds like a good plan!